src/renderer/views/History/History.js

```diff
@@ -44,14 +41,15 @@ export default defineComponent({
       dataLimit: 100,
       searchDataLimit: 100,
       doCaseSensitiveSearch: false,
+      ascending: false,
       showLoadMoreButton: false,
       query: '',
       activeData: [],
     }
   },
   computed: {
     historyCacheSorted: function () {
-      return this.$store.getters.getHistoryCacheSorted
+      return this.ascending ? [...this.$store.getters.getHistoryCacheSorted].reverse() : this.$store.getters.getHistoryCacheSorted
```

Comment:
```suggestion
      return this.ascending ? this.$store.getters.getHistoryCacheSorted.toReversed(): this.$store.getters.getHistoryCacheSorted
```

Reply:
I made the change to use toReversed() with my updates